src/IssueViz.Security/Taint/TaintList/TaintToolWindow.cs

```diff
@@ -52,7 +52,36 @@ public TaintToolWindow(IServiceProvider serviceProvider)
                 componentModel.GetService<ITelemetryManager>()
             );
 
-            Content = new TaintIssuesControl(viewModel);
+            Initialize(viewModel);
+        }
+
+        internal /* for testing */ TaintToolWindow(ITaintIssuesControlViewModel viewModel) =>
```

Comment:
I added a second constructor to make it easier to add some unit tests for the tool window class.